tests/test_legacy.py

```diff
@@ -37,17 +37,17 @@
             "-- CONSISTENT: True",
             "-- DEBUG: True",
             "MATCH (sessions SAMPLE 0.100000)",
-            "SELECT project_id, release, min(max(users) AS max_users) AS min_users",
+            "SELECT min(max(users) AS max_users) AS min_users, project_id, release",
```

Comment:
All of this file is reordering the select columns / conditions to match the order they are processed by the legacy API.